### ISSUE TITLE
Unfriend check throws steam64id undefined error every second

### ISSUE THREAD

**Issue by HerrEurobeat** [HerrEurobeat/steam-comment-service-bot#81]
**Describe the bug**
Unfriend check throws the error down below every second. Doesn't affect the functionality of the bot itself.

**Full error**
```
Uncaught Exception Error! Reason: TypeError: Cannot read property '76561198260031749' of undefined
    at /home/pi/Desktop/steam-comment-service-bot-master/src/controller.js:578:40
    at Array.forEach (<anonymous>)
    at /home/pi/Desktop/steam-comment-service-bot-master/src/controller.js:577:44
    at Array.forEach (<anonymous>)
    at /home/pi/Desktop/steam-comment-service-bot-master/src/controller.js:576:22
    at newArguments.<computed> (/home/pi/Desktop/steam-comment-service-bot-master/node_modules/nedb/lib/executor.js:29:17)
    at Cursor.execFn (/home/pi/Desktop/steam-comment-service-bot-master/node_modules/nedb/lib/datastore.js:484:12)
    at callback (/home/pi/Desktop/steam-comment-service-bot-master/node_modules/nedb/lib/cursor.js:126:19)
    at /home/pi/Desktop/steam-comment-service-bot-master/node_modules/nedb/lib/cursor.js:193:12
    at /home/pi/Desktop/steam-comment-service-bot-master/node_modules/nedb/lib/datastore.js:329:14
```

**Comment by HerrEurobeat** [comment]
Issue found and fixed.  
This issue will stay open until version 2.10.1 containing the fix is published.

**Comment by HerrEurobeat** [comment]
Fixed in f2db506 (Version 2.10.1).
